src/gui/src/components/dashboard-grid/index.tsx

```diff
@@ -176,6 +187,10 @@ export const DashboardGrid = () => {
             </motion.div>
           }
         </AnimatePresence>
+        <Button onClick={onCreateNewProjectClick} className="ml-2">
+          <Plus size={24} />
+          Create New Project
+        </Button>
```

Comment:
This button jumps around/changes position because the switching between the table & card view adds or removes the column filter. I think we should right align the button & that further makes it a standout CTA.


https://github.com/user-attachments/assets/e7d0feeb-efd1-4aec-85e2-49d190fa8520